vendor/github.com/weaveworks/common/server/server.go

```diff
@@ -62,11 +62,12 @@ type Config struct {
 	GRPCListenPort    int    `yaml:"grpc_listen_port"`
 	GRPCConnLimit     int    `yaml:"grpc_listen_conn_limit"`
 
-	HTTPTLSConfig node_https.TLSStruct `yaml:"http_tls_config"`
-	GRPCTLSConfig node_https.TLSStruct `yaml:"grpc_tls_config"`
+	HTTPTLSConfig web.TLSStruct `yaml:"http_tls_config"`
+	GRPCTLSConfig web.TLSStruct `yaml:"grpc_tls_config"`
 
-	RegisterInstrumentation bool `yaml:"register_instrumentation"`
-	ExcludeRequestInLog     bool `yaml:"-"`
+	RegisterInstrumentation  bool `yaml:"register_instrumentation"`
+	ExcludeRequestInLog      bool `yaml:"-"`
+	DisableRequestSuccessLog bool `yaml:"-"`
```

Comment:
We want to disable this in Mimir. We can do it in the same place where we set `ExcludeRequestInLog` too:
https://github.com/grafana/mimir/blob/main/pkg/mimir/mimir.go#L125

Reply:
Thanks @pracucci, will look into it.

Reply:
@pracucci I've disabled it.